### PR TITLE
Update configmap.yaml

### PR DESCRIPTION
**Pull request by MansM**
fixes: https://github.com/netdata/helmchart/issues/140

tested with helm 2 and 3

### CODE REVIEW

File: templates/configmap.yaml [hunk 1]
```diff
@@ -37,7 +37,7 @@ data:
 
 ---
 
-{{- if and .Values.child.enabled .Values.sd.child.enabled .Values.sd.child.configmap.from -}}
+{{ if and .Values.child.enabled .Values.sd.child.enabled .Values.sd.child.configmap.from }}
```
Comment (ilyam8) [inline templates/configmap.yaml:40]: This works, but i think we need to remove only the right delimiter `-`

File: templates/configmap.yaml [hunk 2]
```diff
@@ -17,7 +17,7 @@ data:
 
 ---
 
-{{- if .Values.child.enabled -}}
+{{ if .Values.child.enabled }}
```
Comment (ilyam8) [inline templates/configmap.yaml:20]: same

File: templates/configmap.yaml (hunk 1 above)
Comment (ilyam8) [reply templates/configmap.yaml:40]: otherwise there is extra empty line

**Review by ncmans — APPROVED** [review]

**Review by knatsakis — APPROVED** [review]

**Review by prologic — CHANGES_REQUESTED** [review]
LGTM but @ilyam8 is [right](https://pkg.go.dev/text/template#hdr-Text_and_spaces) Pleaseamend.

**Comment by MansM** [comment]
I am sorry, I am still down with a nasty fever and didnt had time to get to it earlier

**Review by ilyam8 — APPROVED** [review]

**Review by prologic — APPROVED** [review]

**Comment by knatsakis** [comment]
Thank you for your contribution @MansM

**Pull request merged**